schemas/json/block.json

```diff
@@ -245,6 +245,11 @@
 					"description": "This property allows to enable wide alignment for your theme. To disable this behavior for a single block, set this flag to false.",
 					"default": true
 				},
+				"ariaLabel": {
+					"type": "boolean",
```

Comment:
I do not know this area of Gutenberg, but shouldn't this be type string? Unless this simply holds if aria-label is supported or not.

Reply:
Yes, this just allows us to define if `ariaLabel` is _supported_ or not.
I improved the description a bit to make that clearer 👍